docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md

```diff
@@ -0,0 +1,333 @@
+weight: 2
+description: "SOA domains include the deployment of various Oracle Service-Oriented Architecture (SOA) Suite components, such as SOA, Oracle Service Bus (OSB), and Oracle Enterprise Scheduler (ESS)."
+---
+
+#### Contents
+
+* [Limitations](#limitations)
+* [Obtaining the SOA Suite Docker image](#obtaining-the-soa-suite-docker-image)
+* [Creating a SOA Suite Docker image](#creating-a-soa-suite-docker-image)
+* [Configuring access to your database](#configuring-access-to-your-database)
+* [Running the Repository Creation Utility to set up your database schema](#running-the-repository-creation-utility-to-set-up-your-database-schema)
+* [Create a Kubernetes secret with the RCU credentials](#create-a-kubernetes-secret-with-the-rcu-credentials)
+* [Creating a SOA domain](#creating-a-soa-domain)
+
+
+Starting with the 2.2.1 release, the operator supports deployment of SOA Suite components such as Oracle Service-Oriented Architecture (SOA), Oracle Service Bus (OSB), and Oracle Enterprise Scheduler (ESS). Currently the operator supports these different domain types:
+
+* SOA
+* OSB
+* SOAESS
+* SOAOSB
+* SOAESSOSB
+
+This document provides details about the special considerations for running SOA Suite domains with the operator.
+Other than those considerations listed here, SOA Suite domains work in the same way as FMW Infrastructure domains and WebLogic Server domains.
+That is, the remainder of the documentation applies equally to SOA Suite domains.
+
+In this release, SOA Suite domains are supported using the “domain on a persistent volume”
+[model]({{< relref "/userguide/managing-domains/choosing-a-model/_index.md" >}}) only, where the domain home is located in a persistent volume (PV).
+
+
+#### Limitations
+
+Compared to running a WebLogic Server domain in Kubernetes using the operator, the
+following limitations currently exist for SOA Suite domains:
+
+* Domain in image is not supported in this version of the operator.
+* Only configured clusters are supported.  Dynamic clusters are not supported for
+  SOA Suite domains.  Note that you can still use all of the scaling features,
+  you just need to define the maximum size of your cluster at domain creation time.
+  The maximum server size currently supported is 5.
+* SOA Suite domains are not supported with any version of the operator
+  before version 2.2.1.
+* The [WebLogic Logging Exporter](https://github.com/oracle/weblogic-logging-exporter)
+  currently supports WebLogic Server logs only.  Other logs will not be sent to
+  Elasticsearch.  Note, however, that you can use a sidecar with a log handling tool
+  like Logstash or fluentd to get logs.
+* The [WebLogic Monitoring Exporter](https://github.com/oracle/weblogic-monitoring-exporter)
+  currently supports the WebLogic MBean trees only.  Support for JRF MBeans has not
+  been added yet.
+
+
+#### Obtaining the SOA Suite Docker Image
+
+The Oracle WebLogic Server Kubernetes Operator requires patch 29135930.
```

Comment:
Please update this line
The Oracle WebLogic Server Kubernetes Operator requires patch 29135930.
to
The Oracle WebLogic Server Kubernetes Operator requires SOASuite image with patch 29135930.